ssg/products.py

```diff
@@ -26,10 +26,14 @@ def get_all(ssg_root):
         guide_dir = os.path.join(product_dir, product_yaml['benchmark_root'])
         guide_dir = os.path.abspath(guide_dir)
 
-        if 'linux_os' in guide_dir:
-            linux_products.add(product)
-        else:
-            other_products.add(product)
+        relative_extra_rules_dirs = product_yaml.get("extra_rule_group_roots", [])
+        extra_rules_dirs = [os.path.abspath(os.path.join(product_dir, rd)) for rd in relative_extra_rules_dirs]
+
+        for cur_dir in [guide_dir] + extra_rules_dirs:
+            if 'linux_os' in guide_dir:
+                linux_products.add(product)
+            else:
+                other_products.add(product)
```

Comment:
I don't think it makes sense to have a product in multiple categories, I think it should be on the category of the root benchmark.
These categories are used in the `json` utilities and will confuse the tool if a product is assigned to multiple categories.

Reply:
I think that you're right. And, if you look closer, you can find that the code is working exactly like that. I just forgot to revert this file completely because all extended extra_rule_group_roots logic is not needed here. Thanks for noticing, though!